cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -109,6 +110,12 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
         help = "The scanner to use, one of ${Scanner.ALL}."
     ).convertToScanner().default(ScanCode.Factory())
 
+    private val projectScannerFactory by option(
```

Comment:
The commit message seems to assume that dependencies are always third-party dependencies.
Have you considered dependencies which come from the own ORG?
The rationale provided in the commit message suggests to me that a more sophisticated scanner selection mechanism might be needed to enable the use case. What do you think?

Reply:
> Have you considered dependencies which come from the own ORG?

Yes, of course.

> The rationale provided in the commit message suggests to me that a more sophisticated scanner selection mechanism might be needed to enable the use case.

Indeed. This PR is only supposed to be a step into the right direction of a more general scanner selection mechanism as described in https://github.com/oss-review-toolkit/ort/issues/2819.

Reply:
Thanks!